code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -417,13 +419,15 @@
 	interface_desc = "Leave your mark."
 	engage_string = "Toggle stamp type"
 	usable = 1
-	var/iastamp
-	var/deniedstamp
+	var/obj/iastamp			//Theese were just vars, but any device would need to be an object
+	var/obj/deniedstamp //Stops assigning non-objects to theese vars, which probably would break quite a bit.
```

Comment:
@MarsM0nd Yes, I saw that- That's fine, but I'm pointing out that the comment is incorrect. It won't prevent you from changing the variable to a non-object, but will instead just blindly throw a runtime error at that step should there be anything except an object inside of the variable.